src/main/java/org/asteriskjava/manager/event/QueueMemberAddedEvent.java

```diff
@@ -200,17 +199,7 @@ public String getStateinterface()
 	public void setStateinterface(String stateinterface)
 	{
 		this.stateinterface = stateinterface;
-	}
-
-	public String getInterface()
```

Comment:
This looks to be a breaking change. What was the logic behind removing this method?

Reply:
getInterface and setInterface is implemented in AbstractQueueMemberEvent already